### DISCUSSION TITLE
Attach image to email using cid

### DISCUSSION THREAD

**Discussion by Aragur** [pocketbase/pocketbase#4523] [Q&A] [answered]
Hey, 
I want to send custom emails which include a footer with an image.
I saw documentations for other email services (f. e. [Sendgrid](https://sendgrid.com/en-us/blog/embedding-images-emails-facts))  where you can embed an image using a `cid`.
Is this possible in pocketbase? I cannot find anything in the docs.

**Comment by ganigeorgiev** [comment, marked as answer]
The current [`mailer.Message`](https://pkg.go.dev/github.com/pocketbase/pocketbase@v0.22.3/tools/mailer#Message) supports only regular attachments. I'll consider eventually with the refactoring to add an extra `InlineAttachment` field (_or use a special map key modifier_).

For including images in mail templates I usually reference them as absolute links (_the link can point directly to a PocketBase file; if you don't have a specific place where to store the files like `pb_public`, you can create an `assets` collection or similar_).
